### PR TITLE
Blue oyster updates to status monitor

### PR DESCRIPTION
**Pull request by derekpierre**
Based over #74 .

### CODE REVIEW

**Comment by derekpierre** [comment]
Probably needs some backpressure queuing as the network gets bigger, but that can be a separate follow-up PR.

File: monitor/utils.py [hunk 1]
```diff
@@ -47,3 +49,27 @@ def get_etherscan_url(network: str, url_type: EtherscanURLType, address_or_tx_ha
     elif url_type == EtherscanURLType.TRANSACTION:
         # transaction
         return f"https://{url_chain_prefix}etherscan.io/tx/{address_or_tx_hash}"
+
+
+class DelayedLoopingCall(LoopingCall):
```
Comment (KPrasch) [inline monitor/utils.py:54]: :boom:

File: monitor/utils.py [hunk 2]
```diff
@@ -47,3 +49,27 @@ def get_etherscan_url(network: str, url_type: EtherscanURLType, address_or_tx_ha
...
+            raise ValueError("interval must be >= 0")
+        self.running = True
+        # Loop might fail to start and then self._deferred will be cleared.
+        # This why the local C{deferred} variable is used.
+        deferred = self._deferred = defer.Deferred()
+        self.starttime = self.clock.seconds() + self.start_delay
+        self.interval = interval
+        self._runAtStart = now
+        if now:
+            self()
```
Comment (KPrasch) [inline monitor/utils.py:72]: so existential :thinking:

Comment (derekpierre) [reply monitor/utils.py:72]: Haha this is basically a copy from `LoopingCall.start()` with the exception of L68.

**Review by KPrasch — APPROVED** [review]

**Pull request merged**